README.md

```diff
@@ -1,49 +1,51 @@
-# lair-shim
+[![](https://img.shields.io/npm/v/@holo-host/lair-client/latest?style=flat-square)](http://npmjs.com/package/@holo-host/lair-client)
+[![](https://img.shields.io/github/workflow/status/holo-host/lair-client-js/Node.js%20CI/master?style=flat-square&label=master)](https://github.com/holo-host/lair-client-js)
 
-nodejs based shim to route signing requests to lair/wormhole proof-of-concept
+# Lair Client for Node.js
+Javascript implementation of client for Lair Keystore.
 
 
-## Usage
+[![](https://img.shields.io/github/issues-raw/holo-host/lair-client-js?style=flat-square)](https://github.com/holo-host/lair-client-js/issues)
+[![](https://img.shields.io/github/issues-closed-raw/holo-host/lair-client-js?style=flat-square)](https://github.com/holo-host/lair-client-js/issues?q=is%3Aissue+is%3Aclosed)
+[![](https://img.shields.io/github/issues-pr-raw/holo-host/lair-client-js?style=flat-square)](https://github.com/holo-host/lair-client-js/pulls)
 
-1. Launch lair in a terminal like this:
+## Overview
+This module provides APIs for
 
-```
-RUST_LOG=debug lair-keystore -d /tmp/lair
-```
+- connecting to Lair
+- receiving and responding to messages
+- sending requests and awaiting response
+- constructing messages using wire type structs
 
-2. Fire up the the shim:
+### Basic Usage
 
-```
-$ node lair-shim.js
-```
-You should see:
+```javascript
+const { structs, ...lair } = require("@holo-host/lair-client");
 
-```
-Checking for leftover socket.
-No leftover socket found.
-Creating shim server.
-```
+(async () {
+    const client = lair_client.connect( <path to a Lair unix domain socket> );
```

Comment:
where is `lair_client` coming from? Should this be `const client = lair.connect( <path to a Lair unix domain socket> );`
Or line 23 `const { structs, ...lair_client } = require("@holo-host/lair-client");`
?

Reply:
You're correct.  You'd think that having the most basic usage example show accurate code would be a high priority, but not where I come from.  In my world, there are much more important things to contemplate such as the borders of Oceania, or how much wonderflonium is needed for a miniature freeze ray.

```suggestion
    const client = lair.connect( <path to a Lair unix domain socket> );
```

cc @robbiecarlton